packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -114,14 +119,13 @@ export const ProgressIndicatorItem = ({
 					</Text>
 				</Stack>
 
-				{isActive && items?.some((item) => item.isActive) && (
+				{subLevelItem && (
 					<>
 						<ProgressIndicatorItemTimeline />
 
 						<Box as="ul">
 							<Box as="li" css={{ position: 'relative' }}>
 								<Flex
-									{...items.find((item) => item.isActive)}
```

Comment:
I didn't document this very well when I added the level two stuff but we still need to support button. We may need to pick out the `isActive` and `label` and not spread them but allow the rest through?

Reply:
Maybe I should have put a test in 😬

Reply:
I didn't think level 2 supported buttons? Is there still a chance that we need to support other link props though?

Reply:
I have pushed a change to hopefully fix this, but whether it's the best way is up for debate

Reply:
>I didn't think level 2 supported buttons? Is there still a chance that we need to support other link props though?

Yep, you're right on both counts, my bad.

>I have pushed a change to hopefully fix this, but whether it's the best way is up for debate

Yeah, I think this is good.